Rewrite tutorials and templates (#3034)
JetBrains/compose-multiplatform: tutorials/README.md

# Tutorials

## iOS + Android
* [Getting started](https://github.com/JetBrains/compose-multiplatform-ios-android-template)

## Desktop
* [Getting started](https://github.com/JetBrains/compose-multiplatform-desktop-template)
* [Image and icon manipulations](Image_And_Icons_Manipulations)
* [Mouse events and hover](Mouse_Events)
* [Scrolling and scrollbars](Desktop_Components#scrollbars)
* [Tooltips](Desktop_Components#tooltips)
* [Context Menu](Context_Menu/README.md)
* [Top level windows management](Window_API_new)
* [Menu, tray, notifications](Tray_Notifications_MenuBar_new)
* [Keyboard support](Keyboard)
* [Tab focus navigation](Tab_Navigation)
* [Swing interoperability](Swing_Integration)
* [Navigation](Navigation)
* [Accessibility](Accessibility)
* [Building a native distribution](Native_distributions_and_local_execution)

> ## Web

Author: If we have only HTML on this page, people can mistake it with the web target.

Maybe we can write something like this instead?

> The web target is experimental. You can find a demo [here].

P.S. We have `Get started with Compose for Web` on the main page as well

* [Getting started](https://kotl.in/wasm-compose-example)

## General

> See [Foundation](https://developer.android.com/jetpack/compose/documentation#core) and [Design](https://developer.android.com/jetpack/compose/documentation#design) tutorials from Google. They were originally written for Android, but most of information applies for the other targets as well.

Reviewer: ```suggestion
Google's [Foundation](https://developer.android.com/jetpack/compose/documentation#core) and [Design](https://developer.android.com/jetpack/compose/documentation#design) tutorials were originally written for Android but can be used as a reference for other targets.
```

Author: Done

## Libraries

### HTML

> Compose HTML Library allows you to build reactive user interfaces for the web in Kotlin, using the concepts and APIs of Compose to express the state, behaviour, and logic of your application.

Reviewer: ```suggestion
The Compose HTML library allows you to build reactive user interfaces for the web in Kotlin using the concepts and APIs of Compose to express the state, behavior, and logic of your application.
```

Reviewer: Please replace in other places `Compose HTML Library` with `the Compose HTML library`. We should capitalize only the name of the library and use an article.

Author: Done

> HTML Library provides multiple ways of declaring user interfaces in Kotlin code, allowing you to have full control over your website layout with a declarative HTML/CSS API.

Reviewer: ```suggestion
The HTML library provides multiple ways of declaring user interfaces in Kotlin with an ability to fully control your website layout with a declarative HTML/CSS API.
```

Author: Done

> > Note that this library currently supports only Kotlin JS, not Kotlin/Wasm

Reviewer: ```suggestion
> Note that this library doesn't Kotlin/Wasm yet.
```

Reviewer: I would avoid mentioning Kotlin/JS.
